releasenotes/notes/add-parameter-prefix-support-to-ZFeatureMap-ZZFeatureMap-ba13832b9a832e88.yaml

```
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Adds an argument ``parameter_prefix`` to the constructor of ZFeatureMap and
+    ZZFeatureMap classes, to set the prefix of parameters of data encoding 
+    circuit. Fixes #7565. This option gets propagated to parent class 
+    PauliFeatureMap. By default, ``parameter_prefix`` is "x".
```

We can link the actual classes using 
```
:class:`~path.to.Class`
```
and it's good to give an example of what's possible now 🙂 
```suggestion
    Adds an argument ``parameter_prefix`` to the initializer of
    :class:`~qiskit.circuit.library.ZFeatureMap` and
    :class:`~qiskit.circuit.library.ZZFeatureMap`, to set the prefix of parameters
    of the data encoding circuit::
    
        from qiskit.circuit.library import ZFeatureMap
        feature_map = ZFeatureMap(num_qubits=4, parameter_prefix="my_prefix")
```